### PR TITLE
[VSPHERE] Support multiple networks

### PR DESCRIPTION
**Pull request by SvenDowideit**
I'm working on some proxied host testing, so I wanted to use machine to create my proxy server....

```
machine -D create --driver vmwarevsphere \
	--vmwarevsphere-vcenter 10.10.10.2 \
	--vmwarevsphere-boot2docker-url rancheros.iso \
	--vmwarevsphere-network "VM Network" \
	--vmwarevsphere-network internal \
                   dualhome
```

### CODE REVIEW

File: drivers/vmwarevsphere/vsphere.go [hunk 1]
```diff
@@ -325,9 +326,19 @@ func (d *Driver) PreCreateCheck() error {
...
+		// machine assumes there will be a network
+		d.Networks = append(d.Networks, "VM Network")
+	}
+	for _, netName := range d.Networks {
+		if _, err := f.NetworkOrDefault(ctx, netName); err != nil {
+			return err
+		}
 	}
+	// d.Network needs to remain a string to cope with existing machines :/
+	d.Network = d.Networks[0]
```
Comment (shin-) [inline drivers/vmwarevsphere/vsphere.go]: Is this used anywhere else? If not, we may want to set it to empty string instead?

Comment (SvenDowideit) [inline drivers/vmwarevsphere/vsphere.go]: looking at the output of `machine inspect`, I think its better to keep it compatible and fill it with something - to differentiate from a vm with no networking. (and yup, I'm going to be using no-network vm's in future)

Comment (SvenDowideit) [inline drivers/vmwarevsphere/vsphere.go]: it seems that both and empty string, and a csv version makes the provisioning step break - I don't really have the tuits to test all the provisioners, so I'm hoping you'll let this pass for now :)

**Comment by SvenDowideit** [comment]
@shin- ?

**Comment by shin-** [comment]
:+1:

**Pull request merged**